public/preferences.js

```diff
@@ -36,6 +36,12 @@ const defaults = {
   }
 }
 
+let appDataPath = app.getPath('appData')
+
+if (!fs.existsSync(String(appDataPath + '/publish50'))) {
+  fs.mkdirSync(String(appDataPath + '/publish50'))
+}
```

Comment:
Are you sure this is the best way to work around this issue?

Reply:
I'm not quite certain on this one, but publish50 is able to launch even if publish50 folder is not existed in application support folder.

Reply:
Let me look into that.